src/components/githubCTA/index.tsx

```diff
@@ -24,13 +24,16 @@ export function GitHubCTA() {
         welcome, whether fixing a typo (drat!) or suggesting an update ("yeah, this would
         be better").
         <div>
-          <Link href={sourceUrl}>Suggest an edit to this page</Link>{' '}
-          &nbsp;&nbsp;|&nbsp;&nbsp;
-          <Link href="https://docs.sentry.io/contributing/">Contribute to Docs</Link>{' '}
+          <Link href={sourceUrl}>Edit this page in GH</Link> &nbsp;&nbsp;|&nbsp;&nbsp;
```

Comment:
```suggestion
          <Link href={sourceUrl}>Edit this page</Link> &nbsp;&nbsp;|&nbsp;&nbsp;
```
I rarely see GH instead of GitHub so I think the GH looks weird and not everyone will automatically know what it stands for. I also don't think we need to specify that it's github, so maybe we can just leave it out